files/mclass/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "mclass",
+  "repo": "edwinm/mClass",
+  "files": {
+    "include": ["mclass-min.js"]
```

Comment:
This file is not present here https://github.com/edwinm/mClass/tree/1.0.1
The bot will fail